scene/resources/tile_set.cpp

```diff
@@ -1831,7 +1966,7 @@ bool TileSet::_set(const StringName &p_name, const Variant &p_value) {
 			ctd->z_index = p_value;
 
 			// TODO: remove the conversion from here, it's not where it should be done
```

Comment:
Re: the TODO discussion on Rocket.Chat today - you might want to put yourself a reminder in the former of a self-assigned issue ;)

Reply:
It's not really a reminder, it requires reduz's work on a resource conversion system first. So well, I cannot fix it right now. :/
I can replace it by a FIXME if you prefer?

Reply:
Well TODO or FIXME is basically the same ;)

Do we have an issue or proposal that tracks the need for a resource conversion system? If so this can be linked there as a tracker for things that need to be converted.

Reply:
I don't think we have one. I can create one and link it there.
Should I remove the TODO if i link the issue? Or I keep the TODO and add the issue link?

Reply:
We don't have rules for this currently, so do what feels most useful for you, since you'll likely be the one who will have to address this TODO eventually (and ideally it shouldn't stay here for 3 years :D).